gamemodes/nzombies3/gamemode/effects/sv_tesla.lua

```diff
@@ -41,7 +41,7 @@
 --returns: boolean
 --
 
-function nz.Effects.Functions.Tesla( data )
+function Effects:Tesla( data )
```

Comment:
I recommend leaving comments in the code for those who want to understand what it is for trubleshooting reasons :D